### PR TITLE
WindowCovering: Add TC_WNCV_4.5 + update default config

### PR DESCRIPTION
**Pull request by jmeg-sfy**
#### Problem
What is being fixed?  Examples:
* Missing test case for Certif

#### Change overview
* Add missing WNCV 4.5 test to check positions saving
* ONLY on Linux since reboot feature is not available on Darwin
* Zap update all position to zero at first boot
* Zap update Targets to null by default

#### Testing
How was this tested? (at least one bullet point required)
* Test WNCV 4.5 against all-cluster-app

### CODE REVIEW

**Review by jmartinez-silabs — APPROVED** [review]

File: examples/chip-tool-darwin/templates/tests.js [hunk 1]
```diff
@@ -248,18 +248,12 @@ function getTests()
     // some failures on darwin supposely because the ReadClient stays open for the
     // whole duration of the tests and that goes past some internal limits.
     // Because of this, some of the tests are disabled on darwin.
-    'Test_TC_WNCV_1_1',
-    'Test_TC_WNCV_2_1',
-    'Test_TC_WNCV_2_2',
-    'Test_TC_WNCV_2_4',
-    'Test_TC_WNCV_2_5',
+    'Test_TC_WNCV_1_1', 'Test_TC_WNCV_2_1', 'Test_TC_WNCV_2_2', 'Test_TC_WNCV_2_4', 'Test_TC_WNCV_2_5',
```
Comment (bzbarsky-apple) [inline examples/chip-tool-darwin/templates/tests.js:251]: Please leave these one per line.

File: examples/chip-tool-darwin/templates/tests.js [hunk 2]
```diff
@@ -248,18 +248,12 @@ function getTests()
...
     //'Test_TC_WNCV_3_1',
     //'Test_TC_WNCV_3_2',
     //'Test_TC_WNCV_3_3',
-    'Test_TC_WNCV_3_4',
-    'Test_TC_WNCV_3_5',
-    'Test_TC_WNCV_4_3',
-    'Test_TC_WNCV_4_4',
+    'Test_TC_WNCV_3_4', 'Test_TC_WNCV_3_5', 'Test_TC_WNCV_4_3', 'Test_TC_WNCV_4_4',
```
Comment (bzbarsky-apple) [inline examples/chip-tool-darwin/templates/tests.js:255]: Please leave one per line.

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/chip-tool-darwin/templates/tests.js (hunk 1 above)
Comment (jmeg-sfy) [reply examples/chip-tool-darwin/templates/tests.js:251]: The prettier script did this ..

File: examples/chip-tool-darwin/templates/tests.js (hunk 2 above)
Comment (jmeg-sfy) [reply examples/chip-tool-darwin/templates/tests.js:255]: Yep not me, the automated formatter did it....
How to avoid this ?

**Review by ReneJosefsen — APPROVED** [review]

**Pull request merged**